Upgrade logops to 2.1.0

PR https://github.com/telefonicaid/iotagent-manager/pull/159 removed Node v4 from the set of Nodejs supported versions.

We tried at that PR to upgrade logops from 1.0.0 to 2.1.0. However, this causes the "should return the current log level" test in loglevel-api_test.js to fail. In particular, the returned level should be FATAL but it is INFO.

```
{
	"op": "IoTAgentNGSI.DomainControl",
	"time": "2018-11-12T15:24:13.263Z",
	"lvl": "ERROR",
	"msg": "{ AssertionError: expected 'INFO' to be 'FATAL'\n    
  at Assertion.fail (/data/src/iotagent-manager/node_modules/should/lib/assertion.js:92:17)\n    
  at Assertion.value (/data/src/iotagent-manager/node_modules/should/lib/assertion.js:164:19)\n    
  at Request._callback (/data/src/iotagent-manager/test/unit/loglevel-api_test.js:92:41)\n    
  at Request.self.callback (/data/src/iotagent-manager/node_modules/request/request.js:185:22)\n    
  at emitTwo (events.js:126:13)\n    at Request.emit (events.js:214:7)\n    
  at Request.<anonymous> (/data/src/iotagent-manager/node_modules/request/request.js:1161:10)\n    
  at emitOne (events.js:116:13)\n    at Request.emit (events.js:211:7)\n    
  at IncomingMessage.<anonymous> (/data/src/iotagent-manager/node_modules/request/request.js:1083:12)\n    
  at Object.onceWrapper (events.js:313:30)\n    at emitNone (events.js:111:20)\n    
  at IncomingMessage.emit (events.js:208:7)\n    at endReadableNT (_stream_readable.js:1064:12)\n    
  at _combinedTickCallback (internal/process/next_tick.js:139:11)\n    
  at process._tickDomainCallback (internal/process/next_tick.js:219:9)\n  operator: 'to be',\n  expected: 'FATAL',\n  showDiff: true,\n  actual: 'INFO',\n  stackStartFunction: [Function: assert],\n  negate: false,\n  assertion: \n   Assertion {\n     obj: 'INFO',\n     anyOne: false,\n     negate: false,\n     params: \n      { operator: 'to be',\n        expected: 'FATAL',\n        message: undefined,\n        showDiff: true,\n        actual: 'INFO',\n        stackStartFunction: [Function: assert],\n        negate: false,\n        assertion: [Circular] } },\n  _message: 'expected \\'INFO\\' to be \\'FATAL\\'',\n  generatedMessage: true,\n  domain: \n   Domain {\n     domain: null,\n     _events: { error: [Function: domainErrorHandler] },\n     _eventsCount: 1,\n     _maxListeners: undefined,\n     members: [],\n     start: 1542036252453,\n     trans: '1b788697-9989-405f-96d8-1145fc04a7c3',\n     corr: '1b788697-9989-405f-96d8-1145fc04a7c3',\n     op: 'IoTAgentNGSI.DeviceService' },\n  domainThrown: true }"
} 
```

The GET /admin/log operation relies in the IOTAgent Node library, as can be seen in iotagent-manager.js code:

```
...
middleware = require('iotagent-node-lib').middlewares,
...
northboundServer.router.get('/admin/log', middleware.getLogLevel);
```

Not fully sure, but given that the library it still using logops 1.0.0 there is some kind missmatch problem when attempting to use logops 2.1.0 in the IOTAManager.

Let's put this on hold (i.e. let's continue using logops 1.0.0) and once IOTAgent Node Library steps to logops 2.1.0 (by January 2019) try again with the update at IOTAManager.

--- a/CHANGES_NEXT_RELEASE
+++ b/CHANGES_NEXT_RELEASE
@@ -1,0 +1,1 @@
+- Upgrade logops dependency from 1.0.0 to 2.1.0

--- a/package.json
+++ b/package.json
@@ -30,7 +30,7 @@
     "request": "2.88.0",
     "body-parser": "~1.11.0",
     "express": "4.16.4",
-    "logops": "1.0.0",
+    "logops": "2.1.0",
     "underscore": "~1.7.0",
     "revalidator": "~0.3.1",
     "mongoose": "4.13.14",

